test/TEST-41-ONESHOT-RESTART/testsuite.sh

```diff
@@ -18,10 +23,20 @@ TMP_FILE="/test-41-oneshot-restart-test"
 
 touch $TMP_FILE
 
-! systemd-run --unit=two -p StartLimitBurst=3 -p Type=oneshot -p Restart=on-failure -p ExecStart="/bin/bash -c \"printf a >>  $TMP_FILE\"" /bin/bash -c "exit 1"
+# test two: make sure StartLimitBurst correctly limits the number of restarts
```

Comment:
Maybe also include "and restarts execution of the unit from the first ExecStart=" since it is showing that multiple `ExecStart=` properties would not restart from the point that it failed.

Reply:
done!